### ISSUE TITLE
Maintain to user Permission

### ISSUE THREAD

**Issue by pra17shant** [frappe/erpnext#32044]
i have problem or maintain to restrict same user to assign multiple restriction in live server.
i have end user more than 500 and each user have different access and restrictions.
solution to that marked those field convert as "child table" and apply multiple restriction in one screen under targeted user. no need to make duplicate entry, apply new restriction and all think.
so we can maintain easily and professional way hope you consider as a feature request in frappe core level.
![User Permission](https://user-images.githubusercontent.com/99401472/187848851-ee601091-041e-4b1b-971a-cefcb660ccb8.png)

**Comment by creative-paramu** [comment]
Sir, Do you want the below format?
I am attaching the below image for your required development.
![Screenshot from 2022-12-23 16-41-01](https://user-images.githubusercontent.com/101092028/209326207-2d914cfa-2aa7-4d13-bcf6-7ce091b696d5.png)
![Screenshot from 2022-12-23 16-41-38](https://user-images.githubusercontent.com/101092028/209326213-5d1a055e-d99a-4d13-bee1-95691ba20f7e.png)

**Comment by pra17shant** [comment]
> Sir, Do you want the below format? I am attaching the below image for your required development. ![Screenshot from 2022-12-23 16-41-01](https://user-images.githubusercontent.com/101092028/209326207-2d914cfa-2aa7-4d13-bcf6-7ce091b696d5.png) ![Screenshot from 2022-12-23 16-41-38](https://user-images.githubusercontent.com/101092028/209326213-5d1a055e-d99a-4d13-bee1-95691ba20f7e.png)

Exactly! Nice Work... Send Request to Frappe Team for merging Core code

**Comment by creative-paramu** [comment]
> > Sir, Do you want the below format? I am attaching the below image for your required development. ![Screenshot from 2022-12-23 16-41-01](https://user-images.githubusercontent.com/101092028/209326207-2d914cfa-2aa7-4d13-bcf6-7ce091b696d5.png) ![Screenshot from 2022-12-23 16-41-38](https://user-images.githubusercontent.com/101092028/209326213-5d1a055e-d99a-4d13-bee1-95691ba20f7e.png)
> 
> Exactly! Nice Work... Send Request to Frappe Team for merging Core code

@pra17shant , Tq for Your Reply Sir, I Give PR to Frappe.
